docs/tests/navigation/exclude/untitled.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: default
+---
+# Untitled
+
+This page has no `title`, and should not appear in the navigation.
```

Comment:
```suggestion
This page does not have a `title`, and it is excluded from the navigation -- unless it is located in a Jekyll collection (which provides default titles). To exclude a title-less page from the navigation, regardless of whether it is located in a collection, set `nav_exclude: true`.
```

Reply:
@pmarsceill the untitled page appears in the navigation tests when the test pages are moved to a collection, so the original text here was a bit misleading.